ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/filter/role/RoleAuditFilter.java

```diff
@@ -94,7 +105,7 @@ private boolean isWhitelistedUnchecked(AuditEntry logEntry)
         Set<RoleResource> roles = getRoles(logEntry.getUser());
         List<? extends IResource> operationResourceChain = Resources.chain(logEntry.getResource());
         return logEntry.getPermissions().stream()
```

Comment:
It might be a good idea to create a cache on "this level" holding the result (boolen) for the isWhitelisted calculated per (user/resource/permission).

That is - create a cache "key-object" containing values from the logEntry:
* logEntry.getUser()
* logEntry.getResource() - NOT the chain
* logEntry.getPermissions()

And cache the calculation of this calculation:
logEntry.getPermissions().stream()
  .allMatch(permission -> isOperationWhitelistedOnResourceByRoles(permission, operationResourceChain, roles, logEntry.getUser()));